interview-questions.md

```diff
@@ -8,62 +8,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-  Your answer:
-
-  Researched answer:
-
+Your answer: An argument is something that needed in order to log a return to a specific set of information, while a parameters is something that is placed within a function and used as input for our argument
 
+Researched answer: An argument is the actual value, while as a parameter is an input placed within a function
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-  Your answer:
-
-  Researched answer:
-
+Your answer: .map() takes in the predefined parameters from the original variable, the original parameters are required and the new variable is not
 
+Researched answer: Value, Index, and Array. Value is required and index, and array is optional
 
 3. What is the difference between map and filter?
 
-  Your answer:
-
-  Researched answer:
-
+Your answer: map pulls in predefined parameters, while filter allows you to specify what information should be returned
```

Comment:
You can also specify the information to be returned in a map, however, a filter has a built-in if-else statement whereas map you need to define a conditional statement if you wish to.